src/amp-user-notification_with_server_endpoint.html

```diff
@@ -0,0 +1,59 @@
+<!--
+  #### Introduction
+
+  Use amp-user-notification to display a dismissable notification to the user.
```

Comment:
Can we add a sentence about when to use server-side storage, e.g. 'use user-notifications together with a server endpoint if you want to store the dismissal state across your AMP articles and your normal articles.'